app/common/UserAPI.ts

```diff
@@ -1212,6 +1232,20 @@ export class DocAPIImpl extends BaseAPI implements DocAPI {
     return response.data[0];
   }
 
+  public async uploadAttachmentArchive(archive: string | Blob, filename?: string): Promise<ArchiveUploadResult> {
+    const formData = this.newFormData();
+    formData.append('upload', archive, filename);
+    const response = await this.requestAxios(`${this._url}/attachments/archive`, {
+      method: 'POST',
+      data: formData,
+      // On browser, it is important not to set Content-Type so that the browser takes care
+      // of setting HTTP headers appropriately.  Outside browser, requestAxios has logic
+      // for setting the HTTP headers.
```

Comment:
Do you have any links to reference?

Reply:
Did some digging, and found a link which describes why this is important. 
I updated the comment to this:

> // On the browser, Content-Type shouldn't be set as it prevents the browser from setting
> // Content-Type with the correct boundary expression to delimit form fields.
> // https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest_API/Using_FormData_Objects#sending_files_using_a_formdata_object
> // Therefore we omit Content-Type, and allow Axios to handle it as it sees fit - which works
> // correctly in the browser and in Node.